redux_todo/src/actions/index.js

```diff
@@ -0,0 +1,67 @@
+export const addTodo = (value) => {
+  return {
+    type: "ADD_TODO",
+    value,
```

Comment:
conventionally, we try to value as payload:{ value } and use payload.value in the reducer to access the value, this helps in distinguishing between values passed via payload and values that exist in the reducer